utilities/rpp-unittests/HOST/Single_host.cpp

```diff
@@ -0,0 +1,515 @@
+        char temp_second[1000];
+        strcpy(temp_second, src1_second);
+        strcat(temp_second, de->d_name);
+        if (ip_channel == 3)
+        {
+            image = imread(temp, 1);
+            image_second = imread(temp_second, 1);
+        }
+        else
+        {
+            image = imread(temp, 0);
+            image_second = imread(temp_second, 0);
+        }
+        Rpp8u *ip_image = image.data;
+        Rpp8u *ip_image_second = image_second.data;
+        for (j = 0; j < srcSize[i].height * srcSize[i].width * ip_channel; j++)
+        {
+            input[count] = ip_image[j];
+            input_second[count] = ip_image_second[j];
+            count++;
+        }
+        i++;
+        break;
+    }
+
+    closedir(dr2);
+    rppHandle_t handle;
+    rppCreateWithBatchSize(&handle, noOfImages);
+
+    clock_t start, end;
+    double cpu_time_used;
+    start = clock();
+
+    string test_case_name;
+    int test_case = atoi(argv[1]);
+    Rpp32u newMin = 30;
+    Rpp32u newMax = 100;
+    uint kernelSize = 3;
+    Rpp32f alpha = 0.5;
+    Rpp32f beta = 100;
+    Rpp32s adjustmentValue = 100;
+    Rpp32f exposureFactor = 0.5;
+    Rpp32u flipAxis = 0;
+    Rpp32f fogValue = 1;
+    Rpp32f gamma = 0.5;
+    Rpp32f strength = 1.5;
+    Rpp32f zoom = 0;
+    Rpp32f noiseProbability = 0.5;
+    Rpp32f rainPercentage = 0.5;
+    Rpp32u rainWidth = 3;
+    Rpp32u rainHeight = 6;
+    Rpp32f transparency = 0.5;
+    Rpp32f stdDev = 20;
+    Rpp32f snowPercentage = 0.4;
+    Rpp32f angle = 135.0;
+    Rpp32f affine[6] = {1.0, 2.0, 1.0, 1.0, 1.0, 2.0};
+    Rpp32f coordinates[4] = {100, 200, 200, 400};
+    Rpp32f hueShift = 10;
+    Rpp32f saturationFactor = 10;
+    Rpp8u minThreshold = 10;
+    Rpp8u maxThreshold = 30;
+    Rpp32u numOfPixels = 4;
+    Rpp32u gaussianKernelSize = 7;
+    Rpp32f kValue = 1;
+    Rpp32f threshold = 15;
+    Rpp32u nonmaxKernelSize = 5;
+    Rpp32u sobelType = 1;
+    Rpp8u min = 10;
+    Rpp8u max = 30;
+    Rpp32u crop_pos_x = 100;
+    Rpp32u crop_pos_y = 100;
+    Rpp32u xRoiBegin = 50;
+    Rpp32u yRoiBegin = 50;
+    Rpp32u xRoiEnd = 200;
+    Rpp32u yRoiEnd = 200;
+    Rpp32u mirrorFlag = 0;
+
+    Rpp32f perspective[9];
+
+    perspective[0] = 1;
+    perspective[1] = 0;
+    perspective[2] = 0.5;
+    perspective[3] = 0;
+    perspective[4] = 1;
+    perspective[5] = 0.5;
+    perspective[6] = 1;
+    perspective[7] = 0;
+    perspective[8] = 0.5;
+    Rpp32f percentage = 100;
+    Rpp32u numbeoOfShadows = 12;
+    Rpp32u maxSizeX = 12;
+    Rpp32u maxSizey = 15;
+    Rpp32u extractChannelNumber = 1;
+
+    switch (test_case)
+    {
+    case 0:
+        test_case_name = "contrast";
+        rppi_contrast_u8_pkd3_host(input, srcSize[0], output, newMin, newMax, handle);
+        break;
+    case 1:
+        test_case_name = "jitter";
+        rppi_jitter_u8_pkd3_host(input, srcSize[0], output, kernelSize, handle);
+        break;
+    case 2:
+        test_case_name = "blur";
```

Comment:
Codacy found an issue: [Variable 'test_case_name' is assigned a value that is never used.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/rpp/pullRequest?prid=5976402)